### PR TITLE
Feature: QEbatch

### PR DESCRIPTION
**Pull request by kimrojas**
Hi, I'm done with QE batch module

Here is the summary of the changes:
1. Completed Batch implementation for QE
2. Added Comments to describe parts of the code
3. Added Examples for QEbatch

This can be used similar to STATEbatch (with different yaml and csv inputs)

### CODE REVIEW

File: statebatch/statebatch/QEbatch.py [hunk 1]
```diff
@@ -44,16 +63,31 @@ def get_dft_params(atom_to_run):
                 pseudo_input = [element, atomic_masses[atomic_numbers[element]],pseudo_file]
                 pseudo_list.append(pseudo_input)
             input_data['PSEUDOS'] = pseudo_list
+
+            # XC functional preparation
             if input_data["XCTYPE"] in ['vdw-df', 'vdw-df2', 'rev-vdw-df2', 'optb86b-vdw']:
```
Comment (hhlim12) [inline statebatch/statebatch/QEbatch.py]: These lines are for STATE. I think we should remove it for QEbatch

File: statebatch/statebatch/QEbatch.py [hunk 2]
```diff
@@ -44,16 +63,31 @@ def get_dft_params(atom_to_run):
...
                 input_data["VDW-DF"] =  {"QCUT": 10, "NQ": "20"}
```
Comment (hhlim12) [inline statebatch/statebatch/QEbatch.py]: These lines are for STATE. I think we should remove it for QEbatch

File: statebatch/statebatch/QEbatch.py [hunk 3]
```diff
@@ -49,38 +50,52 @@ def get_dft_params(atom_to_run):
...
+            input_data.pop('PSEUDOS', 'not_found')
+
+            input_data['pseudopotentials'] = pseudopotentials # **
+            input_data['PSEUDO_DIR'] = self.comp_spec.get('pseudo_loc')
 
             # XC functional preparation
-            if input_data["XCTYPE"] in ['vdw-df', 'vdw-df2', 'rev-vdw-df2', 'optb86b-vdw']:
-                input_data["VDW-DF"] =  {"QCUT": 10, "NQ": "20"}
+            if {'input_dft', 'vdw_corr'} <=  input_data.keys():
+                raise Exception('Use either input_dft and vdw_corr (not together)')
```
Comment (hhlim12) [inline statebatch/statebatch/QEbatch.py:71]: should be 'either input_dft "**or**" vdw_corr (not together)

File: statebatch/statebatch/QEbatch.py [hunk 4]
```diff
@@ -49,38 +50,52 @@ def get_dft_params(atom_to_run):
...
             return (input_data)
 
         def link():
-            """Links pseudopotential file (not required in QE)"""
+            """Links pseudopotential file and pw (not required in QE)"""
             pwlink = Path(os.path.join(os.getcwd(), self.comp_spec.get('pw_name')))
             pwlink.unlink(missing_ok=True)
             if os.path.exists(pwlink):
                 os.remove(pwlink)
             os.symlink(os.path.join(self.comp_spec.get('pw_loc'), self.comp_spec.get('pw_name')), pwlink)
```
Comment (hhlim12) [inline statebatch/statebatch/QEbatch.py:94]: Why not delete this if it's not required?

**Comment by hhlim12** [comment]
@kimrojas I finished my review. I just have a few comments (see above).

**Comment by kimrojas** [comment]
Thanks! @hhlim12 
QEbatch has been patched based on the review comments.

**Pull request merged**